icons/anonymous.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#111;}</style></defs><title>anonymous</title><path class="cls-1" d="M26.78,27.19l2.14-5.14-2.83,1a8,8,0,0,0,.78-7.16,39.87,39.87,0,0,1,6.81,1.2c-0.85-1-3.13-2.27-6.24-3.16L24.72,7.54A2.37,2.37,0,0,0,22.5,6.25a14.17,14.17,0,0,1-3.21,1.54,17.24,17.24,0,0,1-3.47-1.54,2.36,2.36,0,0,0-2.24,1.34l-2.46,6.32C8,14.8,5.75,16,4.9,17.06a39.87,39.87,0,0,1,6.81-1.2A8,8,0,0,0,12.48,23l-2.81-.95,2.19,5.19a11.77,11.77,0,0,0-4.08,9v0.66h23V36.23A11.81,11.81,0,0,0,26.78,27.19Zm-7.41-1.86-5.16-1.74a7,7,0,0,1-1.37-7.83c2-.17,4.18-0.26,6.45-0.26s4.45,0.09,6.45.26a7,7,0,0,1-1.41,7.87Z"/></svg>
```

Comment:
I need this fill color to be `#111111` rather than `#111` so our color-editing scripts can operate on it, but I can make that change in a following patchset